src/python/grpcio_tests/commands.py

```diff
@@ -125,7 +124,10 @@ def run(self):
         import tests
         loader = tests.Loader()
         loader.loadTestsFromNames(['tests_aio'])
-        runner = tests.Runner()
+        # Even without dedicated threads, the framework will somehow spawn a
+        # new thread for tests to run upon. New thread doesn't have event loop
+        # attached by default, so initialization is needed.
+        runner = tests.Runner(dedicated_threads=False)
```

Comment:
Would it be possible to move all tests to the equivalent of `dedicated_threads=False`? I don't see why we were running them on different threads to begin with.

Reply:
I tested locally, it works fine. I would make the default to be `False`, and keep the code around for a while.

Reply:
I don't want to block this PR because it gives value yes or yes no matter what by reducing complexity. But, we should consider and investigate why current `Aio` tests do not work with having dedicated threads, which IMO should work.

Reply:
I included my deduction in the comments. It works, just require extra call to `set_event_loop` for newly spawned threads. To me, I found existing wrapping of threading not really necessary. A short re-pro of the error:

```Python
import asyncio
import threading

async def work():
    print(f'Hello, from <{threading.current_thread().name}>!')

def start_aio_work():
    asyncio.get_event_loop().run_until_complete(work())

def main():
    # Works fine in main thread
    start_aio_work()
    # Crashes in other threads
    threading.Thread(target=start_aio_work).start()
    

if __name__ == "__main__":
    main()
```
```
Hello, from <MainThread>!
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 917, in _bootstrap_inner
    self.run()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 865, in run
    self._target(*self._args, **self._kwargs)
  File "no_event_loop.py", line 8, in start_aio_work
    asyncio.get_event_loop().run_until_complete(work())
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/asyncio/events.py", line 644, in get_event_loop
    % threading.current_thread().name)
RuntimeError: There is no current event loop in thread 'Thread-1'.
```